easy-ui-react/src/Spinner/Spinner.module.scss

```diff
@@ -0,0 +1,44 @@
+@use "../styles/common" as *;
+
+.spinner {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: design-token("space.1");
+  color: component-token("spinner", "color");
+}
+
+.spinnerIntermediate {
+  height: component-token("spinner", "size");
+  width: component-token("spinner", "size");
+  border: 2px solid;
+  border-bottom-color: design-token("color.neutral.050");
+  border-radius: 50%;
+  display: inline-block;
+  box-sizing: border-box;
+  animation: rotation 2s linear infinite;
```

Comment:
**question**: did we look into how React Spectrum's `ProgressCircle` does their indeterminate state? it looks a bit more dynamic

https://react-spectrum.adobe.com/react-spectrum/ProgressCircle.html

if we want to just spin, might be good to ask rachel about the speed. looks a little slow as-is